--- origsrc/poppler-sharp-0.0.1/Makefile.am	2010-01-30 07:01:13.000000000 -0600
+++ src/poppler-sharp-0.0.1/Makefile.am	2010-06-25 03:01:03.687692400 -0500
@@ -3,6 +3,7 @@
 ACLOCAL_AMFLAGS = -Im4
 
 EXTRA_DIST = \
+	poppler-sharp.dll.config \
 	poppler-sharp/poppler-api.raw \
 	poppler-sharp/poppler.metadata \
 	poppler-sharp/Document.cs
@@ -14,7 +15,7 @@ CLEANFILES = \
 	poppler-sharp/poppler-api.xml \
 	poppler-sharp.dll
 
-pkglib_DATA = poppler-sharp.dll
+pkglib_DATA = poppler-sharp.dll poppler-sharp.dll.config
 pkgconfigdir = $(libdir)/pkgconfig
 pkgconfig_DATA = sources/poppler-sharp.pc
 
--- origsrc/poppler-sharp-0.0.1/poppler-sharp.dll.config	1969-12-31 18:00:00.000000000 -0600
+++ src/poppler-sharp-0.0.1/poppler-sharp.dll.config	2010-06-25 02:59:39.500877200 -0500
@@ -0,0 +1,4 @@
+<configuration>
+	<dllmap dll="poppler-glib" target="libpoppler-glib.so.5" os="!windows,cygwin" />
+	<dllmap dll="poppler-glib" target="cygpoppler-glib-5.dll" os="cygwin" />
+</configuration>
